### PR TITLE
Create mailing-2022-dotnetdaysro.html

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

File: mailing-2022-dotnetdaysro.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+				<tbody>
+					<tr>
+						<td style="padding: 40px 0 20px 0;" class="" valign="top" align="center"><img src="https://go.particular.net/l/27442/2020-06-02/8stf5x/27442/369317/particular_logo_wb.png" style="display: block; width: 209px; height: 50px;" width="209" border="0" height="50"></td>
+					</tr>
+					<!--[if (gte mso 9)|(IE)]> 
+                <tr style="height: 36px"></tr>
+                <![endif]-->
+					<tr>
+						<td style="padding: 0; font-family: Open Sans, Helvetica, Arial, sans-serif;" class="" valign="top" align="center">
+						<h1 style="font-size: 36px; color: #ffffff;">Workshop: Building reliable event-driven microservices architectures</h1>
```
Comment (tmasternak) [inline mailing-2022-dotnetdaysro.html:79]:
```suggestion
						<h1 style="font-size: 36px; color: #ffffff;">Workshop: Building reliable event-driven systems</h1>
```

File: mailing-2022-dotnetdaysro.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+						<p style="color: #bcc9c9; font-size: 20px; line-height: 28px; margin: 0;">Join our colleagues Tomasz Masternak and Szymon Pobiega on<br>
+						Monday and Tuesday, March 21-22nd in an online workshop organized by dotnetdays.ro to learn how to ensure no message is ever lost,<br>
```
Comment (tmasternak) [inline mailing-2022-dotnetdaysro.html:82]:
```markdown
Join [Tomasz Masternak](https://twitter.com/Masternak) and [Szymon Pobiega](https://twitter.com/SzymonPobiega), March 21-22nd, and learn what it takes to build modern, reliable distributed systems that can meet non-trivial business requirements.
In the workshop you'll learn:
*.How to reliably connect the world of synchronous HTTP requests with asynchronous, durable messaging
*.How to make sure consistency between messages processed and business data stored in the system
*.How to design a deduplication strategy for a complex distributed system
Seats are limited so reserve your spot today.
```

File: mailing-2022-dotnetdaysro.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+						<td style="padding: 40px 0 20px 0;" class="" valign="top" align="center"><img src="https://go.particular.net/l/27442/2020-06-02/8stf5x/27442/369317/particular_logo_wb.png" style="display: block; width: 209px; height: 50px;" width="209" border="0" height="50"></td>
+					</tr>
+					<!--[if (gte mso 9)|(IE)]> 
+                <tr style="height: 36px"></tr>
+                <![endif]-->
+					<tr>
+						<td style="padding: 0; font-family: Open Sans, Helvetica, Arial, sans-serif;" class="" valign="top" align="center">
+						<h1 style="font-size: 36px; color: #ffffff;">Workshop: Building reliable event-driven microservices architectures</h1>
+
+						<p style="color: #bcc9c9; font-size: 20px; line-height: 28px; margin: 0;">Join our colleagues Tomasz Masternak and Szymon Pobiega on<br>
```
Comment (tmasternak) [inline mailing-2022-dotnetdaysro.html:81]:
```markdown
Join our colleagues Tomasz Masternak and Szymon Pobiega
March 21-22nd at dotnetdays.ro, for an online workshop and learn how to build reliable, message-based distributed systems in which no message is ever lost, miscommunicated, or duplicated.
```

**Comment by tmasternak** [comment]
@SzymonPobiega I've tweaked the text a bit more. Here is how it looks now:

image

**Comment by SzymonPobiega** [comment]
Awesome ❤️

**Pull request merged**